rules/windows/defense_evasion_execution_windefend_unusual_path.toml

```diff
@@ -32,7 +32,9 @@ process where event.type == "start" and
   (process.name : "MsMpEng.exe" and not
         process.executable : ("?:\\ProgramData\\Microsoft\\Windows Defender\\*.exe",
                               "?:\\Program Files\\Windows Defender\\*.exe",
-                              "?:\\Program Files (x86)\\Windows Defender\\*.exe"))
+                              "?:\\Program Files (x86)\\Windows Defender\\*.exe",
+                              "?:\\Program Files\\Microsoft Security Client\\*.exe",
+                              "?:\\Program Files (x86)\\Microsoft Security Client\\*.exe"))
```

Comment:
Support for Microsoft Security Client ended awhile ago.

Reply:
I see that elastic agent is supported on both Windows 7 and Windows 8.1.

Reply:
The only thing is that Microsoft stopped support for Windows 7 or Windows 8.1 OS as well. But I will tagged @brokensound77 on this one.

Reply:
Tagging @Samirbous as the original rule author, to chime in with his thoughts